### PR TITLE
[Patch] Fix TypeScript typings

### PR DESCRIPTION
**Pull request by Attrash-Islam**
This PR resolves the issue #283 

This bug in typings was introduced accidentally via weird and not correct import at the top of the file in PR #119 

### TODOs:

- [x] Add test to cover such a case.

### CODE REVIEW

File: src/spec/ts.spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { execSync } = require('child_process');
+
+describe('TypeScript Typings', () => {
+    it('Should pass the typings check', () => {
+        execSync('node_modules/.bin/tsc index.d.ts');
+    }).timeout(0);
+});
```
Comment (ealush) [inline src/spec/ts.spec.js:7]: I like to add sanity tests in such cases. The "happy path test" alone doesn't mean much if we don't have a case for it working on an invalid file. How do I know that it really does something and not just exiting immediately?

I would add another test here that checks a failing case in which it doesn't pass the typings check.

Sort of similar to what I mentioned to you [here](https://github.com/Attrash-Islam/react-wisteria/pull/27#discussion_r426073379).

Comment (Attrash-Islam) [reply src/spec/ts.spec.js:7]: Thanks for the review! Do you suggest making a a failing stub similar to `index.d.ts`? Can you elaborate on this

Comment (ealush) [reply src/spec/ts.spec.js:7]: Exactly. The simplest way is to generate a file (`fs.writeFileSync`) in the `before` stage, and clean it up in the `after` stage.

Comment (Attrash-Islam) [reply src/spec/ts.spec.js:7]: I saw that you can import typescript as module and do stuffs with it.. but it's overhead for such a check.

I think of creating a small file that contains typings errors (Don't mind adding it as file without creating it in tests).

Comment (Attrash-Islam) [reply src/spec/ts.spec.js]: ✅

**Pull request merged**